.circleci/config.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - run:
           name: "Set up Cachix"
           command: |
-            nix-env -f $NIXPKGS -iA nixpkgs.cachix nixpkgs.bash
+            nix-env -f $NIXPKGS -iA cachix bash
```

Comment:
To answer the question in the commit message, `nix-env` by default looks at `~/.nix-defexpr` to get the expression to use (with `-A`, the arguments refer to attributes of this expression). `.nix` files get turned into attributes with that name, and subdirectories with `default.nix` get turned into attributes based on the directory name. Typically, there will be a `nixpkgs` directory (which does have a `default.nix`), which is where the `nixpkgs.<attr>` comes from.

However, `-f <file or url>` causes `nix-env` to use the specified path to be used as an expression (for a directory or a URL (which must have a tarball), it uses the default.nix in it as the expression). Since we are pointing directly at nixpkgs when we pass `-f`, we don't need (and also can't specify) the `nixpkgs.` prefix.